ompy/normalizer_gsf.py

```diff
@@ -588,7 +589,11 @@ def fnld(E: ndarray, nld: Vector,
 
     It will take the extrapolation where no exp. data is available.
     """
-    fexp = log_interp1d(nld.E, nld.values)
+    try:
+        fexp = log_interp1d(nld.E, nld.values)
+    except ValueError as e:
+        print(e)
+        raise ValueError("Probably your nld does not extend to 0 MeV. Please see https://github.com/oslocyclotronlab/ompy/issues/170 for more info.")  # noqa
```

Comment:
The exception doesn't trigger when creating the interpolation but when it's evaluated (line 599). A better solution would be to check the validity of interpolation range with the integration range:
```python
if nld.E[0] < np.min(E):
    raise ValueError("Probably your nld does not extend to 0 MeV. Please see https://github.com/oslocyclotronlab/ompy/issues/170 for more info.")
```

Reply:
Although I don't think that it is a major concern, I though of using a `try / except` statement instead of a `if` test due to performance reasons.

Reply:
In case you argue for keeping it as an if test, we might just as well have a assert statement in `Gg_standard`

Reply:
> Although I don't think that it is a major concern, I though of using a `try / except` statement instead of a `if` test due to performance reasons. What do you think?

Probably correct, but then it should be
```python
try:
    return np.piecewise(E, conds, [fexp, nld_model(E[conds[-1]])])
except ValueError as e:
    raise ValueError("Probably your nld does not extend to 0 MeV. Please see https://github.com/oslocyclotronlab/ompy/issues/170 for more info.")  # noqa

Reply:
done